aspnetcore/blazor/components/render-components-outside-of-aspnetcore.md

```diff
@@ -75,14 +74,24 @@ await using var htmlRenderer = new HtmlRenderer(serviceProvider, loggerFactory);
 
 var html = await htmlRenderer.Dispatcher.InvokeAsync(async () =>
 {
-    var parameters = ParameterView.Empty;
-    var output = await htmlRenderer.RenderComponentAsync<Component1>(parameters);
+    var dictionary = new Dictionary<string, object?>
+    {
+        { "Message", "Hello from Render Component Example!" }
+    };
+
+    var parameters = ParameterView.FromDictionary(dictionary);
+    var output = 
+        await htmlRenderer.RenderComponentAsync<RenderComponentExample>(parameters);
```

Comment:
Q: Could this be combined into a single line? Or was it intentionally split to make the line slightly shorter?

Reply:
Shorter line. We have an 85 char limit.

However, I think I'll break it at the dot on the next commit.